trust-store-trust-policy-specification.md

```diff
@@ -1,154 +1,143 @@
 # Trust Store and Trust Policy Specification
 
-This document describes how Notary v2 signatures are evaluated for trust.
+Notary v2 currently supports X.509 based PKI and identities, and uses a trust store and trust policy to determine if a signed artifact is considered authentic.
+
 The document consists of the following sections:
 
-- **[Trust Store](#trust-store)**: Defines set of signing identity that user trusts.
-- **[Trust Policy](#trust-policy)**: Defines how the artifact is evaluated for trust.
+- **[Trust Store](#trust-store)**: Contains a set of trusted identities through which trust is derived for the rest of the system. For X509 PKI, the trust store contains a set of root certificates.
```

Comment:
This is a question.  Is an intermediary Certificate allowed in this collection and thus not always in the true sense a root certificate.

Reply:
Notice below you changed the verbiage to say typically.

Reply:
Yes, intermediate certs are allowed. Updated.
```suggestion
- **[Trust Store](#trust-store)**: Contains a set of trusted identities through which trust is derived for the rest of the system. For X.509 PKI, the trust store typically contains a set of root certificates.
```